modules/tensor_mechanics/test/tests/shell/static/plate_bending.i

```diff
@@ -0,0 +1,199 @@
+# Test for simply supported plate under uniform pressure
+
+# One quarter of a 50 m x 50 m x 1m plate is modeled in this test.
+# Pressure loading is applied on the top surface using nodal forces
+# of magnitude -10 N on all nodes. This corresponds to a pressure (q) of
+# -10.816 N/m^2.
+
+# The FEM solution at (0,0), which is at the center of the full plate
+# is -2.997084e-03 m.
+
+# The analytical solution for displacement at center of plate obtained
```

Comment:
Where did the analytical solution come from? Roark's or something else?

Reply:
I have added the reference for the analytical solution.